README.md

```diff
@@ -10,18 +10,22 @@ az extension add --source $LATEST_RELEASE
 
 # Usage
 
-To list all hanainstances for the subscription:
+To list all HANA instances for the subscription:
 
 ```
 az hanainstance list
 ```
 
-To show details about a specific hana instance:
+To show details about a specific HANA instance:
 
 ```
 az hanainstance show --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME
 ```
 
+To update the Tags field of a specific HANA instance:
+```
+az hanainstance update --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --set tags.newName = newValue
+```
```

Comment:
Is there a way to clear all the tags?

Reply:
if so, can we provide an example?

Reply:
+1

Reply:
I do not know a way yet, I tried `az hanainstance update -g SJCT-T050 -n azlinux0033 --remove tags`. It just send `{}` as the request payload, which according to our design, does nothing.

Reply:
YES! I just figured out a way: `az hanainstance update -g SJCT-T050 -n azlinux0033 --set tags={}` will send `{"tags":{}}` and delete all tags at once